backend/kernelCI_app/routers/databaseRouter.py

```diff
@@ -13,6 +13,9 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
             return db == "cache"
         if hints.get("run_always", False):
             return True
```

Comment:
Also noticed that this flag only applies to the kernelCI_cache app. In normal cases the migrations of kerneci_cache will not be runned for the "dashboard_db" but I think is nice to have a lock on it.
```suggestion
        if hints.get("run_always", False):
            return app_label == "kernelci_cache"
```

Reply:
I mean, it says run _always_, I don't think there's a problem here

Reply:
Alright, I'll change it